src/common/components/repositories-list/index.js

```diff
@@ -62,57 +64,51 @@ class RepositoriesList extends Component {
     }
   }
 
-  onPageLinkClick(pageNumber) {
-    this.props.dispatch(fetchUserRepositories(pageNumber));
-  }
-
   render() {
-    const isLoading = this.props.repositories.isFetching;
-    const pageLinks = this.props.repositories.pageLinks;
+    const isLoading = this.props.snaps.isFetching;
 
     return (
       <div>
-        { this.props.repositories.success && pageLinks &&
-          <div className={ styles['page-links-container'] }>
-            <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
-          </div>
-        }
```

Comment:
I've removed pagelinks because they were built for pagination of results from the GitHub API, not the Launchpad API. Pagination of these results was agreed as a stretch goal.